### PR TITLE
chore(benchmark/camelyon): add new cli arguments

### PR DESCRIPTION
**Pull request by thbcmlowk**
## Companion 

https://github.com/owkin/substra-ci/pull/150

## Related issue

Closes FL-1500

## Please check if the PR fulfills these requirements

- [ ] If the feature has an impact on the user experience, the [changelog](https://github.com/substra/substrafl/blob/main/CHANGELOG.md) has been updated
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] The commit message follows the [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/) specification

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## \[Unreleased\]
 
+### Added
+
+- New CLI arguments to Camelyon benchmark (`--torch-gpu` and `--cp_name`)
```
Comment (ThibaultFy) [inline CHANGELOG.md:13]:
```suggestion
- New CLI arguments to Camelyon benchmark (`--torch-gpu` and `--cp-name`)
```

File: benchmark/camelyon/workflows.py [hunk 2]
```diff
@@ -120,14 +127,15 @@ def accuracy(data_from_opener, predictions):
 
     # Launch experiment
     compute_plan = execute_experiment(
-        client=clients[1],
+        client=clients[0],
```
Comment (ThibaultFy) [inline benchmark/camelyon/workflows.py:130]: Maybe add a comment why we are choosing client 0 ? (I assume it is due to the builder ?)

**Review by ThibaultFy — APPROVED** [review]
Two small comments, LGTM :)

File: CHANGELOG.md (hunk 1 above)
Comment (thbcmlowk) [reply CHANGELOG.md:13]: 🦅

File: benchmark/camelyon/workflows.py (hunk 2 above)
Comment (thbcmlowk) [reply benchmark/camelyon/workflows.py:130]: It is indeed!

File: benchmark/camelyon/workflows.py [hunk 3]
```diff
@@ -40,6 +40,8 @@ def substrafl_fed_avg(
     model: torch.nn.Module,
     credentials_path: Path,
     asset_keys_path: Path,
+    cp_name: str,
```
Comment (guilhem-barthes) [inline benchmark/camelyon/workflows.py:43]:
```suggestion
    cp_name: Optional[str],
```

File: benchmark/camelyon/workflows.py [hunk 4]
```diff
@@ -61,6 +63,8 @@ def substrafl_fed_avg(
         credentials_path (Path): Remote only: file to Substra credentials configuration path.
         asset_keys_path (Path): Remote only: path to asset key file. If un existent, it will be created.
             Otherwise, all present keys in this fill will be reused per Substra in remote mode.
+        cp_name (str): Compute Plan name to display
```
Comment (guilhem-barthes) [inline benchmark/camelyon/workflows.py:66]:
```suggestion
        cp_name (Optional[str]): Compute Plan name to display
```

**Review by guilhem-barthes — COMMENTED** [review]
Thanks for your work! Just really small typing comments

**Pull request merged**